src/domain/orders/details/swap/create.js

```diff
@@ -532,4 +763,54 @@ const SwapMenu = ({ order, onCreate, onDismiss, toaster }) => {
   )
 }
 
+const CurrencyInput = ({ value, currency, ...props }) => {
```

Comment:
I dont know if it's convention but would it make sense to group these with the rest of the styled components at the top of the file